Dockerfile

```diff
@@ -1,170 +1,86 @@
 FROM ubuntu:22.04 as base
 WORKDIR /workdir
 
+ARG sdk_nrf_branch=v2.4-branch
+ARG toolchain_version=v2.4.1
+ARG sdk_nrf_commit
+ARG NORDIC_COMMAND_LINE_TOOLS_VERSION="10-23-0/nrf-command-line-tools-10.23.0"
 ARG arch=amd64
-ARG crossarch=arm-zephyr-eabi
-ARG ZEPHYR_TOOLCHAIN_VERSION=0.16.0
-ARG ZEPHYR_TOOLCHAIN_ARCHIVE_FORMAT=xz
-ARG WEST_VERSION=1.0.0
-# These are the legacy utils, see https://github.com/NordicPlayground/nrf-docker/issues/68
-ARG NRF_UTIL_VERSION=6.1.7
-ARG NORDIC_COMMAND_LINE_TOOLS_VERSION="10-21-0/nrf-command-line-tools-10.21.0"
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-# System dependencies
-RUN mkdir /workdir/.cache && \
-    apt-get -y update && \
-    apt-get -y upgrade && \
-    apt-get -y install \
-        wget \
-        python3-pip \
-        python3-venv \
-        ninja-build \
-        gperf \
-        git \
```

Comment:
Hi, I don't know if this is the right place for pointing this out, but it seems like `git` is not installed on the docker image. My colleagues and I are experiencing failed builds using both github actions or bitbucket pipelines due to
```
git: command not found
```
Maybe this is the reason? Last Friday things were working fine. Thank you in advance!